### ISSUE TITLE
consider adding Resource Owner Password Credentials Grant flow

### ISSUE THREAD

**Issue by michielbdejong** [remotestorage/spec#83]
http://tools.ietf.org/html/rfc6749#section-4.3
This might be useful for desktop and device-native apps. It would open up the way for adding remoteStorage support to https://github.com/owncloud/android-library and https://github.com/cozy/cozy-mobile

**Comment by michielbdejong** [comment]
Maybe next time

**Comment by michielbdejong** [comment]
Maybe next time. In any case, servers can give the option to generate a bearer token for people who want to configure automated scripts without interactive GUI, where implicit grant cannot be used.

**Comment by skddc** [comment]
Also, I'd like to see PIN flow before password flow, so that apps never see a (much more powerful) account password. :)

**Comment by untitaker** [comment]
Do we really want this in draft-07? Proposing "maybe one day" milestone.

**Comment by skddc** [comment]
We actually support redirect flow in Cordova in rs.js these days (which would cover the Cozy app it seems), and ownCloud can do the same using an in-app browser window. Revealing your account password to apps is not really required for auth in native/mobile apps.

**Comment by untitaker** [comment]
Then I suppose we agree?

BTW having a "whenever" milestone is IMO semantically equivalent to not having
any milestone.

**Comment by skddc** [comment]
Yes. I tried to signal agreement via the thumbs-up on your comment.

**Comment by skddc** [comment]
Proposing to close this issue for now, because afaics nobody could bring a strong enough argument for why redirect flow isn't enough. WDYT?
